config/initializers/govuk_formbuilder.rb

```diff
@@ -3,8 +3,8 @@
   #
   # https://www.rubydoc.info/gems/govuk_design_system_formbuilder/GOVUKDesignSystemFormBuilder
 
-  # config.brand: 'govuk'
-  #
+  config.brand = "nhsuk"
```

Comment:
@tvararu does this potentially open the door to using the form builder in lots more places?

Reply:
@benilovj I think so... I haven't tested this in anger so I don't know if it works for every single component or whether some form elements are different. We'll have to see.

Reply:
Unfortunately, I've been looking, but the same can't be easily done for `govuk-components`. Some of the NHS components have a different HTML structure anyway, so simply changing the CSS class wouldn't make much of a difference.